test/commands/templates/install.test.js

```diff
@@ -14,6 +14,32 @@ const TheCommand = require('../../../src/commands/templates/install')
 const BaseCommand = require('../../../src/BaseCommand')
 const { TEMPLATE_PACKAGE_JSON_KEY, readPackageJson, writeObjectToPackageJson, getNpmDependency } = require('../../../src/lib/npm-helper')
 
+// mock project-installation calls
+jest.mock('@adobe/aio-lib-console-project-installation')
+const TemplateHandler = require('@adobe/aio-lib-console-project-installation')
+const mockTemplateHandlerInstance = {
+  installTemplate: jest.fn()
+}
+TemplateHandler.init.mockResolvedValue(mockTemplateHandlerInstance)
+
+// mock app config calls
+jest.mock('@adobe/aio-cli-lib-app-config')
+const loadConfig = require('@adobe/aio-cli-lib-app-config')
+const mockAIOConfigJSON = JSON.parse('{"aio": {"project": {"id": "project-id","org": {"id": "org-id"}}}}')
+loadConfig.mockImplementation(() => { return mockAIOConfigJSON })
```

Comment:
nitpick, as this is an arrow function expression you can use
```suggestion
loadConfig.mockImplementation(() => mockAIOConfigJSON)
```

Reply:
Done :white_check_mark: